### DISCUSSION TITLE
Projects Beta archived tasks: how to unarchive

### DISCUSSION THREAD

**Discussion by keesvanbemmel** [community/community#11158] [Projects and Issues]
Hi!

We'e been using Projects Beta now for a while. After archiving some tasks we don't have a clue how to show/see/unarchive them.

Any thoughts?

Kees.

**Comment by frederic-mamath** [comment]
I'm also looking to view archived ticket. Is there a way to see them ? Are they saved somewhere ?

**Comment by marcus-ahle** [comment]
Running into this as well. The cards just...disappeared from the project entirely.
